src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala

```diff
@@ -1579,5 +1579,24 @@ class AppsResourceTest extends AkkaUnitTest with GroupCreation {
       (appJson \ "fetch" \ 0 \ "cache" get) should be(JsBoolean(false))
       (appJson \ "fetch" \ 0 \ "destPath" get) should be(JsString("bash.copy"))
     }
+
+    "Allow creating app with network name with underscore" in new Fixture {
+      Given("An app with a Docker config.json")
+      val container = RamlContainer(
+        `type` = EngineType.Mesos,
+        docker = Option(DockerContainer(
+          image = "private/image")))
```

Comment:
...and `ctrl-c ctrl-v` again?

Reply:
yeah, sorry, embarassing :( I rushed this PR to get it still done on Friday. Thanks for catching that...